<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_FocusMode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {border:1px solid gray; border-radius:5px; margin-top:1ex;}QGroupBox::title{subcontrol-origin:margin; subcontrol-position:top left; padding:0 0px;}</string>
    </property>
    <property name="title">
     <string>Focus Mode</string>
    </property>
    <widget class="QPushButton" name="pushButton_FocusApply">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>85</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_Direct">
        <property name="text">
         <string>Direct</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_Infinity">
        <property name="text">
         <string>Infinity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_Macro">
        <property name="text">
         <string>Macro</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>264</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_AF_On">
        <property name="text">
         <string>Auto Focus ON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_AF_Off">
        <property name="text">
         <string>Auto Focus OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>251</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Focus Position(0-1023):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_FocusPosition"/>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget1</zorder>
    <zorder>layoutWidget2</zorder>
    <zorder>layoutWidget3</zorder>
    <zorder>pushButton_FocusApply</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_OIS_Mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {border:1px solid gray; border-radius:5px; margin-top:1ex;}QGroupBox::title{subcontrol-origin:margin; subcontrol-position:top left; padding:0 0px;}</string>
    </property>
    <property name="title">
     <string>OIS Mode</string>
    </property>
    <widget class="QWidget" name="layoutWidget4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>324</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_OIS_Mode0">
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_OIS_Mode3">
        <property name="text">
         <string>Movie</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_OIS_Mode1">
        <property name="text">
         <string>Zero shutter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButton_OIS_Mode4">
        <property name="text">
         <string>High SR Movie</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_OIS_Mode2">
        <property name="text">
         <string>Exposure/Shake eval.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton_OIS_Mode5">
        <property name="text">
         <string>View Finder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_OIS_Apply">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>85</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_LSC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>401</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>LSC (Lens Shading Correction)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_StillCapture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {border:1px solid gray; border-radius:5px; margin-top:1ex;}QGroupBox::title{subcontrol-origin:margin; subcontrol-position:top left; padding:0 0px;}</string>
    </property>
    <property name="title">
     <string>Still Capture</string>
    </property>
    <widget class="QWidget" name="layoutWidget5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_Still12M_NML">
        <property name="text">
         <string>12M Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Still12M_HDR">
        <property name="text">
         <string>12M HDR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Highspeed">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {border:1px solid gray; border-radius:5px; margin-top:1ex;}QGroupBox::title{subcontrol-origin:margin; subcontrol-position:top left; padding:0 0px;}</string>
    </property>
    <property name="title">
     <string>Highspeed</string>
    </property>
    <widget class="QPushButton" name="pushButton_Highspeed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>23</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
